radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java

```diff
@@ -280,14 +278,25 @@ public void commit(AID aid) {
 		}
 	}
 
+	private LedgerEntryStoreResult doStore(
+			byte prefix,
+			LedgerEntry ledgerEntry,
+			LedgerEntryIndices indices,
+			Transaction transaction) throws IOException {
+		byte[] ledgerEntryData = encode(ledgerEntry);
+		log.debug("Storing atom {} with size {}", ledgerEntry.getAID(), ledgerEntryData.length);
```

Comment:
Probably should be trace level, rather than debug.

Reply:
Done